graph/list.go

```diff
@@ -53,7 +62,8 @@ func (s *TagStore) CmdImages(job *engine.Job) error {
 	if err != nil {
 		return err
 	}
-	lookup := make(map[string]*engine.Env)
+
+	lookup := make(map[string]*types.Image)
```

Comment:
Why you using pointers here if you anyway converting all of them to value below, when construct `images` slice.